client/coniksclient/internal/cmd/run.go

```diff
@@ -0,0 +1,186 @@
+package cmd
+
+import (
+	"log"
+	"net/url"
+	"os"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run the test client.",
+	Long: `run gives you a REPL, so that you can invoke commands to perform CONIKS operations including registration and key lookup. Currently, run supports:
+	- register:
```

Comment:
How do you feel about this?

```
register [name] [key]
```

because I keep typing,

```
lookup [name]
```

by accident.

Also, there should be some feedback for unrecognized commands.